packages/ipfs/test/core/libp2p.spec.js

```diff
@@ -8,6 +8,8 @@ const Libp2p = require('libp2p')
 const EE = require('events')
 const libp2pComponent = require('../../src/core/components/libp2p')
 
+const isBrowser = require('is-browser')
```

Comment:
No need for external lib, `isBrowser` is already implemented by `ipfs-utils`:

```suggestion
const { isBrowser } = require('ipfs-utils/src/env')
```